cdap-ui/Gulpfile.js

```diff
@@ -289,6 +288,7 @@ gulp.task('watch:js:app:babel', function() {
     .pipe(plug.babel({
       presets: ['es2015']
     }))
+    .pipe(plug.ngAnnotate())
```

Comment:
Any specific reason why move this down? I thought we are ng-annotating things before we send it to babel. Babel adds/pollyfills a lot of stuff and we are annotating the code after that. Thought it might be better to annotate and send it to babel.